usl_pipeline/cloud_functions/requirements.txt

```diff
@@ -1,16 +1,25 @@
 affine==2.4.0
 attrs==23.2.0
+basemap==1.3.9
+basemap-data==1.3.2
+black==24.4.2
 blinker==1.7.0
 cachetools==5.3.3
 certifi==2024.2.2
+cftime==1.6.4
 charset-normalizer==3.3.2
 click==8.1.7
 click-plugins==1.1.1
 cligj==0.7.2
 cloudevents==1.10.1
+contourpy==1.2.1
+cycler==0.12.1
 deprecation==2.1.0
 fiona==1.9.6
+flake8==7.1.0
```

Comment:
it looks like dev tools like flake8 & MyPy are in here. Let's just keep this to the actual production requirements.

Reply:
Done!